### DISCUSSION TITLE
cannot deactivate System.err logs

### DISCUSSION THREAD

**Discussion by MLobatoR** [antlr/antlr4#4741] [General]
Hi,

I think you might forgot to comment System.err all over your code. I don't want to check them while running, just during debug. Is there a way that I can disable this?

Best regards,
Miriam Lobato

**Comment by kaby76** [comment]
Java or Dart? Which calls? Like the one in here? https://github.com/antlr/antlr4/blob/67228355c5bfd1ed5ebb89e726992ec43dda7b53/runtime/Java/src/org/antlr/v4/runtime/ConsoleErrorListener.java#L31-L39

**Reply by MLobatoR** [reply]
Hi @kaby76,

Java, And yes, like that one.

**Comment by jimidle** [comment]
You need to remove the default error listeners and add your own, which can do anything it wishes.

```java
	parser.removeErrorListeners();
	parser.addErrorListener( errorListener );
```

https://www.antlr.org/api/Java/org/antlr/v4/runtime/ANTLRErrorListener.html

https://www.aldoraweb.com/antlr-error-handle/

**Reply by MLobatoR** [reply]
@jimidle I'll try that.

Thank you for your help.
